question-service/migrations/20220910135435-create-question.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Questions', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      q_name: {
+        type: Sequelize.STRING
+      },
```

Comment:
Consider standardizing the naming format, use camelCase since the rest are all using camelCase as well

Reply:
@Hzxin perhaps can address this early to use camelCase, since the convention in JS is camelCase. Other than this LGTM for me